apps/taiga-lumbermill/src/dashboards/dashboards/dashboards.component.html

```diff
@@ -11,6 +12,19 @@
     <span tuiNavigationLogo>
         <span tuiFade>{{ title }} dashboard</span>
     </span>
-    <hr />
+    <tui-tabs
+        tuiFade
+        [activeItemIndex]="0"
```

Comment:
after reloading page wrong tab is active. You should use routerLinkActive directive instead of activeItemIndex